docker-mender-convert

```diff
@@ -25,13 +25,21 @@ MENDER_CONVERT_DIR="$(pwd)"
 #Pass in version in-case we are added as a git submodule
 MENDER_CONVERT_VERSION=$(git_mender_convert_version)
 
+# Create a unique work directory so we can run multiple containers in parallel
+WORK_DIR=$(mktemp -d -p $(pwd) -t work.XXXX)
+WORK_SUFFIX=$(echo $WORK_DIR | awk -F. '{print $NF}')
+
 docker run \
   --rm \
   -v $MENDER_CONVERT_DIR:/mender-convert \
+  -v $WORK_DIR:/mender-convert/work \
   --privileged=true \
   --cap-add=SYS_MODULE \
   -v /dev:/dev \
   -v /lib/modules:/lib/modules:ro \
   --env MENDER_ARTIFACT_NAME=${MENDER_ARTIFACT_NAME} \
   --env MENDER_CONVERT_VERSION=${MENDER_CONVERT_VERSION} \
   $IMAGE_NAME "$@"
+
+mv ${WORK_DIR}/convert.log ./convert.log.${WORK_SUFFIX}
+sudo rm -rf ${WORK_DIR}
```

Comment:
This step can be done inside docker instead to avoid the need for sudo.

Reply:
The _contents_ of WORK_DIR can be removed but not work.XXXX in the parent.